client/src/app/+video-channels/video-channels-routing.module.ts

```diff
@@ -1,10 +1,21 @@
 import { NgModule } from '@angular/core'
 import { RouterModule, Routes } from '@angular/router'
+import { MyVideoChannelCreateComponent } from './video-channel-edit/video-channel-create.component'
+import { VideoChannelUpdateComponent } from './video-channel-edit/video-channel-update.component'
 import { VideoChannelPlaylistsComponent } from './video-channel-playlists/video-channel-playlists.component'
 import { VideoChannelVideosComponent } from './video-channel-videos/video-channel-videos.component'
 import { VideoChannelsComponent } from './video-channels.component'
 
 const videoChannelsRoutes: Routes = [
+  {
+    path: '@create',
```

Comment:
`'create'`

Reply:
That would collide with a channel named "create"

Reply:
:o Nice catch

Then maybe `/manage/create` and `/manage/update/:videoChannelName`?

Reply:
Isn't it better with `/c/@create` and `/c/@update/:videoChannelName` then?

Reply:
It's just you introduce this `@` which is a good idea but not consistent with the rest of the project

I would expect to also have `/my-library/video-playlists/@create` which is not the case and I don't want to update all these routes :grin: